private/model/api/operation.go

```diff
@@ -126,6 +132,9 @@ func (c *{{ .API.StructName }}) {{ .ExportedName }}Request(` +
 //
 {{ end -}}
 {{ end -}}
+{{ if not $blacklisted -}} 
+// Please also see ` + redirectsURL + `/goto/WebAPI/{{ $.API.Metadata.UID }}/{{ .ExportedName }}
```

Comment:
`redirectURL` should be added to the template data not injected into the string like this.